src/sentry/snuba/metrics/fields/base.py

```diff
@@ -953,6 +958,22 @@ def run_post_query_function(
                 org_id=org_id, metric_ids=metric_ids, alias=alias
             ),
         ),
+        SingularEntityDerivedMetric(
+            metric_mri=TransactionMRI.ALL_USER.value,
+            metrics=[TransactionMRI.USER.value],
+            unit="percentage",
+            snql=lambda *_, org_id, metric_ids, alias=None: uniq_aggregation_on_metric(
+                metric_ids, alias=alias
+            ),
+        ),
+        SingularEntityDerivedMetric(
+            metric_mri=TransactionMRI.USER_MISERY.value,
+            metrics=[TransactionMRI.MISERABLE_USER.value, TransactionMRI.ALL_USER.value],
+            unit="percentage",
+            snql=lambda miserable_user, user, org_id, metric_ids, alias=None: division_float(
+                addition(miserable_user, MISERY_ALPHA), addition(user, MISERY_BETA), alias
```

Comment:
If I read the discover code correctly, this should be

```suggestion
                addition(miserable_user, MISERY_ALPHA), addition(user, MISERY_ALPHA + MISERY_BETA), alias
```

reference:

https://github.com/getsentry/sentry/blob/23d11f0077bc4ed617c7b170074d1f6e2e4fb612/src/sentry/search/events/datasets/discover.py#L1202

https://github.com/getsentry/sentry/blob/23d11f0077bc4ed617c7b170074d1f6e2e4fb612/src/sentry/search/events/datasets/discover.py#L211

Reply:
Good catch! will fix